backend/contactsAPI.py

```diff
@@ -39,6 +43,28 @@ def get_contacts():
         return Response(status=204)
 
 
+@app.route('/img', methods=['POST'])
+def image():
+    if request.method == 'POST':
+        token = request.headers.get('token')
+        _id = request.form.get('_id')
+        user = User().find_by_token(token)
```

Comment:
Doesn't look like `user` is getting used at all, can you confirm the contact id is in the users contact list?

Reply:
This looks better